### PR TITLE
Migration of SIT-2878 template-sap2wday-organization-broadcast to Mule 4

### PR DESCRIPTION
**Pull request by mariandug**

### CODE REVIEW

File: doc/configReadme.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
++ sap.jco.operationtimeout `1000`
++ sap.jco.connectioncount `2`
++ sap.jco.gwhost `your.sap.addres.com`
++ sap.jco.gwservice `sapgw14`
++ sap.jco.idoc.programid `PROGRAM_ID`
+
+**Workday Connector configuration**
+
++ wday.user `user`
```
Comment (ICfl) [inline doc/configReadme.xml]: should be username

File: doc/configReadme.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
++ wday.password `secret`
++ wday.endpoint `https://impl-cc.workday.com/ccx/service/mulesoft_pt1/Human_Resources/v23.1`
```
Comment (ICfl) [inline doc/configReadme.xml]: wday.endpoint is not in property file

File: doc/configReadme.xml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
++ wday.system.id `System id`
```
Comment (ICfl) [inline doc/configReadme.xml]: system.id is not consistent with property in implementation

File: src/main/resources/mule.dev.properties [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+#SAP Endpoint configuration
+sap.jco.operationtimeout=
+sap.jco.connectioncount=
+sap.jco.gwhost=
+sap.jco.gwservice=
+sap.jco.idoc.programid=
+
+# Workday Connector configuration
+wday.username=
+wday.tenant=
```
Comment (ICfl) [inline src/main/resources/mule.dev.properties]: wday.tenant is missing in documentaion

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+            <version>${sap.connector.version}</version>
+            <classifier>mule-plugin</classifier>
+        </dependency>
+        <!-- Please Add dependencies for SAP libraries -->
+        <dependency>
+            <groupId>com.mulesoft.connectors</groupId>
+            <artifactId>mule-workday-connector</artifactId>
+            <classifier>mule-plugin</classifier>
+        </dependency>
+    </dependencies>
```
Comment (ICfl) [inline pom.xml]: missing repositories

File: doc/configReadme.xml (hunk 2 above)
Comment (mariandug) [reply doc/configReadme.xml]: removed from doc

File: doc/configReadme.xml (hunk 3 above)
Comment (mariandug) [reply doc/configReadme.xml]: fixed in doc

File: src/main/resources/mule.dev.properties (hunk 4 above)
Comment (mariandug) [reply src/main/resources/mule.dev.properties]: added to doc

File: pom.xml (hunk 5 above)
Comment (mariandug) [reply pom.xml]: added to pom.xml

**Pull request merged**